### ISSUE TITLE
Support Jenkins themes in Franklin

### ISSUE THREAD

**Issue by logankilpatrick** [tlienart/Franklin.jl#767]
I know this may not be possible or even useful to most, but given how popular Jenkins is, it would seemingly make sense to support in some way Jenkins themes.

**Comment by tlienart** [comment]
Right so I should probably pin an issue somewhere but for new themes basically the procedure is as follows:

* someone suggests a theme they would like to use
* we provide a draft port of the theme
* we ask the someone to use the port and ping back and forth with issues so that we can polish the theme together
* theme is released for everyone

There's three currently "draft-mode" theme like this:

- https://github.com/tlienart/FranklinTemplates.jl/issues/63
- https://github.com/tlienart/FranklinTemplates.jl/issues/112
- https://github.com/tlienart/FranklinTemplates.jl/issues/116

Unfortunately the requesters seem quite busy so for now these are a bit slow.

In short: porting theme is relatively easy to do, I'm happy to help, but I do want people to help back. If someone wants a new theme or to start a website with a given theme, I will help them *quite a lot* but the counterpart is that they help me polish the port for others to use.

**Comment by logankilpatrick** [comment]
You're a rockstar, thank you! I'll wait until I need this more urgently but can definitely help at some point.
